Lib/_pyio.py

```diff
@@ -40,6 +40,29 @@
 _CHECK_ERRORS = _IOBASE_EMITS_UNRAISABLE
 
 
+def text_encoding(encoding, stacklevel=2):
+    """
+    A helper function to choose the text encoding.
+
+    When encoding is not None, just return it.
+    Otherwise, return the default text encoding (i.e. "locale").
+
+    This function emits an EncodingWarning if *encoding* is None and
+    sys.flags.encoding_warning is True.
```

Comment:
```suggestion
    sys.flags.warn_default_encoding is True.
```

Was this mistakenly not updated to the new name?